tests/v2/validation/snapshot/snapshot_recurring_restores_test.go

```diff
@@ -0,0 +1,77 @@
+//go:build (validation || extended || infra.any || cluster.any) && !sanity && !stress
```

Comment:
we haven't talked about a strategy for this yet, but I think having separate files for each static test will be overkill since all of the setup is the same. How would we feel about:
* if:
  * the setup is the same (all non-test functions/types) and
  * the go tags are the same
* then: 
  * the Tests would go in the same file

this will help cut down on copy code.

another point about having separate files for everything is, it wouldn't make sense to have dynamic tests in these files since they are all doing the same thing across the suite they're in.

Reply:
It's a bit tricky IMO because in this PR as an example, we're checking for replacing worker nodes and recurring restores. I don't necessarily think we should have those together in one test file.

I do agree with the dynamic tests, I will remove those.

Reply:
Spoke offline with the team regarding this. We agreed as a best practice that we will consolidate functional areas in one file. I will make this change.

Reply:
Resolved.